src/Rings/groebner.jl

```diff
@@ -162,12 +167,34 @@ function groebner_basis_with_transform(B::BiPolyArray; ord::Symbol = :degrevlex,
   return BiPolyArray(B.Ox, i), map_entries(x->B.Ox(x), m)
 end
 
-function lift_groebner_basis(I::MPolyIdeal, ord::Symbol; complete_reduction::Bool=false)
+
+@doc Markdown.doc"""
+    groebner_basis_with_transform(I::MPolyIdeal; ord::Symbol = :degrevlex, complete_reduction::Bool=false)
+
+Given an ideal `I` and optional parameters `ord` for a monomial ordering and `complete_reduction`
+this function computes a Groebner basis (if `complete_reduction = true` the reduced Groebner basis) of the
+ideal spanned by the elements in `B` w.r.t. the given monomial ordering `ord` and the
+transformation matrix from the ideal to the Groebner basis.
```

Comment:
Several remarks:
- There is no `B` anymore 
- That `I` is an ideal is already clear from the documented type
- so is that ord and complete_reduction are optional types
- it is not clear to me what a "transformation matrix from an ideal to ..." something is -- I assume it is from `gens(I)` to the GB?

To be constructive, here's a suggestion how it could perhaps be written (which is overall more in line how other documentation in Julia and OSCAR is written, I think):
```suggestion
Return a pair `G, m` where `G` is a Groebner basis of the ideal `I` with respect to the
monomial ordering `ord`, and `m` is a transformation matrix from `gens(I)` to `GB`.
If `complete_reduction` is set to `true` then `G` will be the reduced Groebner basis.
```